### PR TITLE
docs: add only 1.20- for xcuitest in real devices

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

The bootstrap.sh is already removed.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/drivers/ios-xcuitest-real-devices.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ installed, and is _not_ recommended):
 ```
 *   Given this installation location, `/path/where/installed/bin/appium`, `WebDriverAgent` project
     will be found in `/path/where/installed/lib/node_modules/appium/node_modules/appium-webdriveragent`.
-    Open a terminal and go to that location, then run the following in order to
+*  (Only for lower than Appium 1.20) Open a terminal and go to that location, then run the following in order to
```
Comment (mykola-mokhnach) [inline docs/en/drivers/ios-xcuitest-real-devices.md:133]: Only for Appium versions below 1.20

Comment (mykola-mokhnach) [reply docs/en/drivers/ios-xcuitest-real-devices.md:133]: Also we still need to create the bundle folder, only the second step should be skipped

File: docs/en/drivers/ios-xcuitest-real-devices.md [hunk 2]
```diff
@@ -130,10 +130,12 @@ installed, and is _not_ recommended):
 ```
 *   Given this installation location, `/path/where/installed/bin/appium`, `WebDriverAgent` project
     will be found in `/path/where/installed/lib/node_modules/appium/node_modules/appium-webdriveragent`.
-    Open a terminal and go to that location, then run the following in order to
-    set the project up:
+    Open a terminal and got to the location, then run the next script.
 ```
     mkdir -p Resources/WebDriverAgent.bundle
+```
+*  (Only for Appium version below 1.20) Run the next script on the same terminal.
```
Comment (mykola-mokhnach) [inline docs/en/drivers/ios-xcuitest-real-devices.md:137]: version -> versions

File: docs/en/drivers/ios-xcuitest-real-devices.md [hunk 3]
```diff
@@ -130,10 +130,12 @@ installed, and is _not_ recommended):
 ```
 *   Given this installation location, `/path/where/installed/bin/appium`, `WebDriverAgent` project
     will be found in `/path/where/installed/lib/node_modules/appium/node_modules/appium-webdriveragent`.
-    Open a terminal and go to that location, then run the following in order to
-    set the project up:
+    Open a terminal and got to the location, then run the next script.
```
Comment (mykola-mokhnach) [inline docs/en/drivers/ios-xcuitest-real-devices.md:133]: go to the location

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by Thuytien** [comment]
Hi @KazuCocoa : After I updated appium v1.21.0 I got this error: An unknown server-side error occurred while processing the command. Original error: Unable to launch WebDriverAgent because of xcodebuild failure. Could you pls help to guide me how can I fix it? Thanks in advance.

**Comment by KazuCocoa** [comment]
Please create a new issue with `appium:showXcodeLog` capability logs

**Comment by Thuytien** [comment]
@KazuCocoa : thank you, I just found you guide about webdriveragent and followed it. My issue is solved. Thank you.

**Pull request merged**